release/benchmarks/distributed/test_many_actors.py

```diff
@@ -76,4 +83,5 @@ def no_resource_leaks():
                 "perf_metric_type": "THROUGHPUT",
             }
         ]
+    dashboard_test.update_release_test_result(results)
```

Comment:
Q: Should we fail the scalability test when the latency is too high? Or should we just check it regularly from the nightly?